### DISCUSSION TITLE
R12DS

### DISCUSSION THREAD

**Discussion by sukhoi222** [pascallanger/DIY-Multiprotocol-TX-Module#949] [General]
I am trying to bind R12DS rx from Radiolink with my Radiomaster TX16S and didn´t manage to get it work. Same with other DS receivers from Radiolink. Anyone knows how to?
Thks

**Comment by pascallanger** [comment]
Supported Radiolink RXs are listed in the doc: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#radiolink---74
